client.js

```diff
@@ -1410,59 +1410,74 @@ class Client {
   #########################################################################
   */
 
-  async getHistoricalData(p) {
-    /*
-    Requests contracts' historical data. When requesting historical data, a
-    finishing time and date is required along with a duration string.
-
-    requestId:TickerId - The id of the request. Must be a unique value. When the
-        market data returns, it whatToShowill be identified by this tag. This is also
-        used when canceling the market data.
-    contract:Contract - This object contains a description of the contract for which
-        market data is being requested.
-    endDateTime:str - Defines a query end date and time at any point during the past 6 mos.
-        Valid values include any date/time within the past six months in the format:
-        yyyymmdd HH:mm:ss ttt
-
-        where "ttt" is the optional time zone.
-    duration:str - Set the query duration up to one week, using a time unit
-        of seconds, days or weeks. Valid values include any integer followed by a space
-        and then S (seconds), D (days) or W (week). If no unit is specified, seconds is used.
-    barSizeSetting:str - Specifies the size of the bars that will be returned (within IB/TWS listimits).
-        Valid values include:
-        1 sec
+  /*
```

Comment:
Isn't `/**` ?

Reply:
Could be `/**` then `*` prefixing each comment line, or just `/*` for now.

Initially I didn't want to put in too much effort in case my PR wasn't accepted (like the TypeScript proposal).

Reply:
For now, moving the comment before the function call would add it IntelliSense.

Reply:
I see. AFAIK it's not JSDocs compilant: https://jsdoc.app/about-getting-started.html
but for sure doesn't make things worse :)

So will merge now.
Thanks for contribution.